### PR TITLE
US 1974660: Previous versions cleanup - part 40

### PR DESCRIPTION
**Pull request by craigcaseyMSFT**
Project to clean up the previous versions for "Driver Reference" articles. Moving a couple of files for "usbcon" technology.

### CODE REVIEW

**Comment by craigcaseyMSFT** [comment]
Hi @mhopkins-msft  - Please review and let me know if this is OK to merge. Thanks much!

File: wdk-ddi-src/content/ucxroothub/nf-ucxroothub-ucx_roothub_config_init_with_control_urb_handler.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## -description
+
+Initializes a [**UCX\_ROOTHUB\_CONFIG**](ns-ucxroothub-_ucx_roothub_config.md) with pointers to event callback functions implemented by the client driver.
+
+## -parameters
+
+### -param Config
+
+[out] A pointer to a [**UCX\_ROOTHUB\_CONFIG**](ns-ucxroothub-_ucx_roothub_config.md) structure to initialize.
```
Comment (mhopkins-msft) [inline wdk-ddi-src/content/ucxroothub/nf-ucxroothub-ucx_roothub_config_init_with_control_urb_handler.md]: The [in] and [out] designators should go on the same line with the param. It should look like this:

### -param Config [out]

A pointer to a [**UCX\_ROOTHUB\_CONFIG**](ns-ucxroothub-_ucx_roothub_config.md) structure to initialize.

**Review by mhopkins-msft — CHANGES_REQUESTED** [review]
The [in] and [out] designators should go on the same lines with params. See inline comment.

**Comment by craigcaseyMSFT** [comment]
> 

Thanks, @mhopkins-msft for your feedback. I actually had the following instruction from Lori on this and we've done all of them like I did in this PR. This was the instruction I received:

"Meriwether will complain if SAL is on the ### -param line. Change all of these to be formatted as follows:
       -param Context
       [in] A pointer to ..."

@lorihollasch  - can you please confirm? Thanks much!

**Comment by lorihollasch** [comment]
@craigcaseyMSFT , go ahead and make the change that @mhopkins-msft  has requested. I just checked Meriwether and it does not seem to be consistent in its complaining, sorry!

**Review by mhopkins-msft — APPROVED** [review]

**Comment by craigcaseyMSFT** [comment]
Everything reviewed and approved.

#sign-off

**Comment by mhopkins-msft** [comment]
Thanks @craigcaseyMSFT!

**Pull request merged**